lib/environment.rb

```diff
@@ -438,14 +438,18 @@ def nodes(user: admin, refresh: false, quiet: true)
     #   support purposes (e.g. host a debug pod for running node commands)
     def service_project
       unless @service_project
-        project = Project.new(name: "tests-" + EXECUTOR_NAME.downcase, env: self)
+        # if the cluster set the default scheduler, set the project running debug pod node-selector=''
+        # to overwrite the default scheduler, or the pod can not be run successfully
+        project_name = "tests-" + EXECUTOR_NAME.downcase
+        project = Project.new(name: project_name, env: self)
         unless project.active?
           # 30 seconds is no longer enough
           project.wait_to_disappear(admin, 60)
           res = project.create(by: admin, clean_up_registered: true)
           unless res[:success]
             raise "failed to create service project #{project.name}, see log"
           end
+          admin.cli_exec(:annotate, resource: "namespace", resourcename: project_name, keyval: 'openshift.io/node-selector=', overwrite: true)
```

Comment:
Will it affect the other projects ? What if they rely on it ?

Reply:
Thanks for the review! This is only for project `test- + EXECUTOR_NAME.downcase`, if I understand correctly, this ns is used for `oc debug` cmd only.

Reply:
Could you run some test with your branch to make sure it does not break the other testing ?

Reply:
Cause Jenkins and Polarshift are not stable, run some test cases from my localhost.
```
$ git branch
  OCP-11971
  OCP-9929
  aws-fix
  iscsi
  local-stoage
  local-storage
  master
* scheduler
  v3

cucumber features/storage/persistent_volume.feature |tee test.log

    [04:13:02] INFO> Shell Commands: oc delete projects --all --kubeconfig=/home/piqin/workdir/preserve-storage-server1-piqin/ocp4_testuser-21.kubeconfig
    project.project.openshift.io "qu2ib" deleted
    
    [04:13:09] INFO> Exit Status: 0
    [04:13:09] INFO> waiting up to 30 seconds for user clean-up to take place
    [04:13:10] INFO> REST delete_oauthaccesstoken for user 'BushSlicer::APIAccessor:testuser-21@ocp4', base_opts: {:options=>{:api_version=>"v1", :accept=>"application/json", :content_type=>"application/json", :oauth_token=>"sha256~MmdB_DrvSx1l53D-XSIwfG8RhaxZk8QfqWSKPDWJ4sU"}, :base_url=>"https://api.piqin-0512.qe.devcluster.openshift.com:6443", :headers=>{"Accept"=>"<accept>", "Content-Type"=>"<content_type>", "Authorization"=>"Bearer <oauth_token>"}}, opts: {:token_to_delete=>"sha256~4fRvw1OUuDtIjgR8QUKavyUKXA2lrSu89erNMZDx2A8"}
    [04:13:10] INFO> HTTP DELETE https://api.piqin-0512.qe.devcluster.openshift.com:6443/apis/oauth.openshift.io/v1/oauthaccesstokens/sha256~4fRvw1OUuDtIjgR8QUKavyUKXA2lrSu89erNMZDx2A8
    [04:13:10] INFO> HTTP DELETE took 0.686 sec: 200 OK | application/json 242 bytes
    
    [04:13:10] INFO> Shell Commands: rm -r -f -- /home/piqin/workdir/preserve-storage-server1-piqin
    
    [04:13:11] INFO> Exit Status: 0
    [04:13:12] INFO> === End After Scenario: PV can not bind PVC with mismatched accessMode, Examples (#3) ===

10 scenarios (10 passed)
111 steps (111 passed)
6m27.695s